modules/overviews/app/components/project_custom_fields/sections/edit_component.html.erb

```diff
@@ -0,0 +1,41 @@
+<%=
+  component_wrapper do
+    primer_form_with(
+      model: @project,
+      method: :put,
+      data: { turbo: true, turbo_stream: true, "test-selector": "async-dialog-content" },
+      url: update_project_custom_values_path(project_id: @project.id, section_id: @project_custom_field_section.id),
+    ) do |f|
+      component_collection do |collection|
+        collection.with_component(Primer::Alpha::Dialog::Body.new(my: 3, classes: "Overlay-body_autocomplete_height")) do
```

Comment:
We do not need the `Dialog::Body` here any more, as it is now correctly set by the Dialog. Currently, this results in additional outer spacing:

<img width="400" alt="Bildschirmfoto 2024-04-18 um 11 20 09" src="https://github.com/opf/openproject/assets/7457313/8be55a4f-c0cf-41df-a81d-c834fa1e62e6">